package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "Jiffy Web",
   "author": "Shubham Singh <imshubhamsingh@gmail.com>",
-  "license": "GNU",
```

Comment:
why ?

Reply:
i have not changed these files, may be auto generated